### PR TITLE
Update some useful tool 

### PR DESCRIPTION
**Pull request by yunxijun**
### Types of Changes

- [x] New addition to list
- [ ] Fixing bug in existing item in list
- [ ] Removing item from list
- [ ] Changing structure (organization) of list

### Proposed Changes

1. Added some useful software recommended by many people

### PR Checklist

- [x] I have read the CONTRIBUTING guide
- [x] I have explained why this PR is important
- [x] I have added necessary documentation (if appropriate)

### Further Comments

- Thanks to share it.
- N/A

### CODE REVIEW

File: README-zh.md [hunk 1]
```diff
@@ -799,7 +800,7 @@ Awesome Mac
 * [Streaker](https://github.com/jamieweavis/streaker) - GitHub贡献和统计跟踪菜单栏应用程序。 [![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/jamieweavis/streaker)
 * [The Unarchiver](https://theunarchiver.com/) - 解压许多不同种类的归档压缩文件。 [![Open-Source Software][OSS Icon]](https://bitbucket.org/kosovan/theunarchiver) ![Freeware][Freeware Icon]
 * [Timing](https://timingapp.com/) - Mac 的自动时间和生产力跟踪。
-* [Unarchive One](https://cleanerone.trendmicro.com/unarchiver-one/) - 快速解压单个多个不同种类的压缩文件/压缩文件到各类场景压缩格式。 ![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://apps.apple.com/cn/app/unarchiver-one-rar-zip-tool/id1127253508?l=en&mt=12)
+* [Unarchive One](https://cleanerone.trendmicro.com/unarchiver-one/?utm_source=github&utm_medium=referral&utm_campaign=githubproject) - 快速解压单个多个不同种类的压缩文件/压缩文件到各类常见压缩格式。 ![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://apps.apple.com/app/apple-store/id1127253508?pt=444218&ct=GitHub&mt=8)
```
Comment (jaywcjlove) [inline README-zh.md:803]: https://apps.apple.com/app/apple-store/id1127253508?pt=444218&ct=GitHub&mt=8
Can't open in browser

Comment (yunxijun) [reply README-zh.md:803]: @jaywcjlove U.S. region can be opened

File: README-zh.md [hunk 2]
```diff
@@ -842,9 +843,11 @@ Awesome Mac
...
+* [AdBlock One](https://cleanerone.trendmicro.com/ad-block-one-for-mac/?utm_source=github&utm_medium=referral&utm_campaign=githubproject) - 适用于MacOS/iOS的免费广告拦截器 停止在Safari中看到烦人的广告。更快地打开网站。更安全地浏览网页。![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://apps.apple.com/app/apple-store/id1491889901?pt=444218&ct=GitHub&mt=8)
 * [AppCleaner](http://freemacsoft.net/appcleaner/) - 一个小应用程序，让你彻底卸载不需要的应用程序。![Freeware][Freeware Icon]
 * [AppTrap](http://onnati.net/apptrap/) - 删除APP的同时移除文件。 [![Open-Source Software][OSS Icon]](https://github.com/kvijayan/AppTrap) ![Freeware][Freeware Icon]
 * [blueutil](https://github.com/toy/blueutil) - 命令行蓝牙控制工具，可以配合[SleepWatcher](https://www.bernhard-baehr.de/)实现MacBook合盖瞬间关闭蓝牙，开盖自动打开蓝牙。这在使用蓝牙耳机时尤其有用。[![Open-Source Software][OSS Icon]](https://github.com/toy/blueutil) ![Freeware][Freeware Icon]
+* [Cleaner One](https://apps.apple.com/app/apple-store/id1133028347?pt=444218&ct=GitHub&mt=8) - 多合一磁盘清理管理器：清理您的 Mac 并优化其性能，立即运行快速扫描以验证什么占用了您的存储空间。![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://apps.apple.com/app/apple-store/id1133028347?pt=444218&ct=GitHub&mt=8)
```
Comment (jaywcjlove) [inline README-zh.md:850]: https://apps.apple.com/app/apple-store/id1133028347?pt=444218&ct=GitHub&mt=8

Can't open in browser

Comment (yunxijun) [reply README-zh.md:850]: @jaywcjlove This is the case with hi. We mainly provide links in the United States. If you are not in China, you can open it. Of course, in China, you can search directly in the AppStore.

File: README-zh.md (hunk 1 above)
Comment (jaywcjlove) [reply README-zh.md:803]: in China => `https://apps.apple.com/cn/app/`

```diff
- https://apps.apple.com/app/
+ https://apps.apple.com/cn/app/
```

Comment (yunxijun) [reply README-zh.md:803]: @jaywcjlove Hi, for some reason, the link provided here should be a foreign link, is this okay? Or is it possible to provide two links here, one for domestic and one for foreign countries

Comment (jaywcjlove) [reply README-zh.md:803]: `README-zh.md` -> Chinese users. `https://apps.apple.com/cn/app/` Can be opened in a browser

Comment (yunxijun) [reply README-zh.md:803]: @jaywcjlove Ok i am going to change it now

Comment (yunxijun) [reply README-zh.md]: @jaywcjlove Ok I adjust it. Thanks

**Pull request merged**